extensions/agent/src/dialogs/alertDialog.ts

```diff
@@ -334,14 +563,18 @@ export class AlertDialog extends AgentDialog<AlertData> {
 			if (index >= 0) {
 				severityNumber = index + 1;
 			}
+		} else {
+			let errorNumber = +this.errorNumberTextBox.value;
+			if (errorNumber) {
+				severityNumber = errorNumber;
+			}
 		}
 		return severityNumber;
 	}
 
 	protected updateModel() {
 		this.model.name = this.nameTextBox.value;
 		this.model.isEnabled = this.enabledCheckBox.checked;
-
 		this.model.alertType = this.getDropdownValue(this.typeDropDown);
 		this.model.databaseName = this.getDropdownValue(this.databaseDropDown);
 		this.model.severity = this.getSeverityNumber();
```

Comment:
I merge a few changes to this file.  Ping me if you want to go through the merge conflicts together.